src/home/examples.md

```diff
@@ -21,27 +21,46 @@ Housing Cooperatives: Divine House, [Art Cooperative](https://www.ic.org/artcomm
+Cooperative Store / Gallery: [Dutch Alley Artists Co-op](http://www.dutchalleyartistsco-op.com/aboutus.html), [A.I.R Gallery](https://www.airgallery.org), [Ujamaa Collective](http://www.ujamaacollective.org/)_, [ARTZ (Ancestral Rich Treasures of Zuni) Cooperative](https://zunipuebloart.com/artz-co-op)_, [Qualla Arts & Crafts,](https://quallaartsandcrafts.com)\_ [Art Center Cooperative](http://www.tacjacksonville.org/about/), [Aarhus Makers](https://aarhusmakers.com/), [Ulična galerija ](https://www.ulicnagalerija.rs)
 
 Cooperative Studios: [Talking Dolls](http://talkingdollsdetroit.com/), [Adaept](https://adaept.design)
 
 Cooperative Recording Spaces: [Live Musicians Co-op](https://www.livemusicianscoop.com/?fbclid=IwAR3lVIM63gnYMMwMFcEYdlb1ZnVm0yUW4LkoOD0NxhUcceEYKZdnc8qKMgg)
 
-Cooperative Darkrooms: [Lone Star Darkroom](https://www.lonestardarkroom.com), [Bushwick Community Darkroom](https://www.bushwickcommunitydarkroom.com) [](http://www.clayartpotterygroup.com/about-us.html)
+Cooperative Darkrooms: [Lone Star Darkroom](https://www.lonestardarkroom.com), [Bushwick Community Darkroom](https://www.bushwickcommunitydarkroom.com), [Clay Art Pottery Co-op](http://www.clayartpotterygroup.com/about-us.html)
 
 Cohousing and Intentional Communities: [Convent Arts Community](https://www.ic.org/artcommunes/), [MilePost 5](https://www.ic.org/artcommunes/)
 
-Cooperative Co-working, Retreat, Residency, or #Landback Network: [ZEAL](https://www.policylink.org/node/62766)_, [Activation Residency](https://activationresidency.com/)_, [Flux Factory](https://www.fluxfactory.org), [The Weavers Project](https://www.theweaversproject.org)_, [Soul Fire Farm](https://www.soulfirefarm.org)_, [Yo Mama’s House](https://www.yomamashouse.com)_, [Black \[Space] Residency](https://www.blackspaceresidency.com),_ [Land Relationships Super Collective\*](http://www.landrelationships.com/introduction) (\* = majority BIPOC membership)
+Cooperative Co-working, Retreat, Residency, or Landback Network: [ZEAL](https://www.policylink.org/node/62766)_, [Activation Residency](https://activationresidency.com/)_, [Flux Factory](https://www.fluxfactory.org), [The Weavers Project](https://www.theweaversproject.org)_, [Soul Fire Farm](https://www.soulfirefarm.org)_, [Yo Mama’s House](https://www.yomamashouse.com)_, [Black \[Space Residency](https://www.blackspaceresidency.com)_, [Land Relationships Super Collective](http://www.landrelationships.com/introduction)\_
 
+_\_ = majority BIPOC membership_
 :::
 
-::: break-inside-avoid
+::: list-item
```

Comment:
I can't tell what this does?

Reply:
Thank you for pointing it out! I used for a section that was then removed and forgot to remove the markdown. It's removed now.

Reply:
Be sure to also remove this from [.eleventy.js](https://github.com/jboolean/art-coop/pull/72/files#diff-c306e0a99961a16f5c5c83996caa0958b94006d97f97475049ea3a08036bb5b0)

Reply:
removed, thank you for the reminder.